packages/db/test/cli/start.test.mjs

```diff
@@ -256,3 +256,17 @@ test('start with hotreload disabled', async ({ equal, same, match, teardown }) =
   match(url, /http:\/\/127.0.0.1:[0-9]+/)
   child.kill('SIGINT')
 })
+
+test('start with hotreload enabled', async ({ equal, same, match, teardown }) => {
+  const db = await connectAndResetDB()
+  teardown(() => db.dispose())
+
+  await db.query(db.sql`CREATE TABLE pages (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(42)
+  );`)
+
+  const { child, url } = await start('-c', join(import.meta.url, '..', 'fixtures', 'simple.json'), '--hot-reload', true)
+  match(url, /http:\/\/127.0.0.1:[0-9]+/)
+  child.kill('SIGINT')
+})
```

Comment:
@marcopiraccini my OS is Ubuntu 20.04.5 LTS and Node is 18, would be nice if you can double check me on this PR since in my env. test and this one are falling  https://github.com/platformatic/platformatic/pull/679#issuecomment-1427927338

Reply:
OK, tested. I also uses Ubuntu (well, PopOS, but it's the same :)) and node 18 (v18.12.1). I tested your branch and I got the same problem. Then I tested with your `main` (which is behind of some commits) and I had the SAME problem. 
So, the issue is note related with you PR, and I guess you can solve it locally just getting latest commits and running `pnpm install`. 
Note that the error is in a dependency, so my guess is that it has been already solved (I tested in current `main` and the test passes).

Reply:
Thank you very much @marcopiraccini PopOS UI is really beautiful, I hope one day I will try it :) 

```
 guess you can solve it locally just getting latest commits and running pnpm install
```
Tests are now passing in my environment :100: 
Thank you for sharing knowledge with me :)